gb-dbg/src/debugger/status_bar.rs

```diff
@@ -0,0 +1,37 @@
+use crate::dbg_interfaces::{CpuRegs, DebugOperations};
+use egui::Ui;
+
+const Z_FLAG: u16 = 0b1000_0000;
+const N_FLAG: u16 = 0b0100_0000;
+const H_FLAG: u16 = 0b0010_0000;
+const C_FLAG: u16 = 0b0001_0000;
+
+pub struct StatusBar;
+
+impl StatusBar {
+    pub fn draw<DBG: DebugOperations>(&self, ui: &mut Ui, regs: &DBG) {
+        ui.horizontal(|ui| {
+            ui.label(format!(
+                "Flags: {}",
+                display_flags(u16::from(regs.cpu_get(CpuRegs::AF)))
+            ));
+        });
+    }
+}
+
+fn display_flags(af_reg: u16) -> String {
+    let mut f_display: Vec<char> = vec!['_'; 4];
```

Comment:
I suggest to replace `_` with `-`

Reply:
I tried and felt it didn't look good sadly :/